src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+
+export default class AddItemForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      input: ""
+    };
+  }
+  onUpdate = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    this.props.addItem(this.state.input);
+    this.setState({ input: "" });
+  };
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <input
```

Comment:
You can add the word `required` to any input to prevent empty submissions. It's the same as writing `required="true"`